vcbuild.bat

```diff
@@ -197,9 +197,12 @@ if defined msi (
     goto wix-not-found
   )
 )
-@rem VS2015 vsvarsall is quick, so run anyway
+
+@rem check if VS2015 is already setup
+if "_%VisualStudioVersion%_" == "_14.0_" if "_%VCVARS_VER%_" == "_140_" goto :found_vs2015
```

Comment:
The earlier similar line for 2017 doesn't prefix the goto target with a colon. Please can you make these consistent?